src/component/tags/create-tag-container.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import TagComponent from './form-tag-component';
+import { create } from '../../store/tag/actions';
+
+const mapStateToProps = () => ({
+    tag: { id: -1, type: '', value: '' },
+});
+
+const mapDispatchToProps = dispatch => ({
+    submit: tag => create(tag)(dispatch),
+});
```

Comment:
you can actually do: 

```js
const mapDispatchToProps = {
    create,
};
``` 

....and react-redux will acutally do the dispatch magic for you.